StudCity.Frontend/.env

```diff
@@ -1,2 +1,3 @@
 REACT_APP_BASE_API_URL=https://stud-city-api.azurewebsites.net/
-REACT_APP_CHAT_HUB_URL=https://stud-city-api.azurewebsites.net/chatHub/
\ No newline at end of file
+REACT_APP_CHAT_HUB_URL=https://stud-city-api.azurewebsites.net/chatHub/
+REACT_APP_GOOGLE_CLIENT_ID=361271152420-l9ggnenveihf9u1a9q91b2uj8qsgfec6.apps.googleusercontent.com
```

Comment:
is it okay to show such thing in the env file?

Reply:
Nothing critical, this key i`m using in all my apps. So when we go in production i set else client id key